### ISSUE TITLE
bug with comments

### ISSUE THREAD

**Issue by talgautb** [postcss/sugarss#12]
I have such problem.
input:
```
.page
	font-family: 'Open Sans', Arial, serif
	/* transform: rotate(90deg) */
	background: black
```
output:
```
.page {
	font-family: 'Open Sans', Arial, serif
	/* transform: rotate(90deg) */
	background: black {}
}
```

**Comment by ai** [comment]
I will look tomorrow. Is it OK?

**Comment by ai** [comment]
You can use inline comments for now `//`

**Comment by talgautb** [comment]
@ai ok, i test it for myself. with `//` same problem

**Comment by ai** [comment]
Fixed 1452e9a
